File: PHP/Orang.php
<?php
class Orang {
    protected $nik;
    protected $nama;
    protected $jenis_kelamin;

    public function __construct($nik, $nama, $jenis_kelamin) {
        $this->nik = $nik;
        $this->nama = $nama;
        $this->jenis_kelamin = $jenis_kelamin;
    }

    public function get_nik() { return $this->nik; }
    public function get_nama() { return $this->nama; }
    public function get_jenis_kelamin() { return $this->jenis_kelamin; }
}
?>

File: JAVA/Orang.java
public class Orang {
    private String nik;
    private String nama;
    private String jenisKelamin;

    // Constructor kosong (opsional tapi baik untuk kebiasaan)
    public Orang() {}

    // Constructor dengan parameter
    public Orang(String nik, String nama, String jenisKelamin) {
        this.nik = nik;
        this.nama = nama;
        this.jenisKelamin = jenisKelamin;
    }

    // Getter
    public String getNik() { return nik; }
    public String getNama() { return nama; }
    public String getJenisKelamin() { return jenisKelamin; }
}